app/src/main/cpp/Instead/CMakeLists.txt

```diff
@@ -35,9 +36,10 @@ set( SOURCES
         "${INSTEAD_SRC_DIR}/SDL2_rotozoom.c"
         "${INSTEAD_SRC_DIR}/SDL_anigif.c"
         "${INSTEAD_SRC_DIR}/SDL_gfxBlitFunc.c"
-        "${INSTEAD_SRC_DIR}/noise1234.c" )
+        "${INSTEAD_SRC_DIR}/noise1234.c"
+        "${INSTEAD_SRC_DIR}/guniprops.c" )
```

Comment:
There is no such file in the INSTEAD sources.

Reply:
As I understood this file is part of Glib. I still don't want to include it in the application. You wrote that it can be replaced by HarfBuzz. Maybe it's better to do this?

Reply:
@btimofeev unfortunately it turned out that harfbuzz alone can not do that (i asked on their maling list).